docs/analysis/criteria.md

```diff
@@ -174,48 +206,31 @@ Alternatively, files should be brought into the website repo via
+  reason?
+
 ### Minimal website requirements
 
 Listed here are the _minimal_ website requirements for projects based on their
-[maturity level][]: sandbox, incubating, graduated and archived.
+[maturity level]: sandbox, incubating, graduated and archived.
 
 Except for archived projects, requirements are cumulative through project
-maturity levels so, for example, incubating projects must satisfy the
-requirements for sandbox projects.
-
-- **Sandbox**
-  - [Website guidelines](../website-guidelines-checklist.md): a majority of the
-    guidelines are satisfied
-  - **Project documentation** may or may not be present -- it is acceptable at
-    this maturity level to link out to documentation that hasn't yet been
-    integrated into the website
-    - _Example_: website with a single homepage, without any documentation or,
-      as was mentioned above, linking out to an external (preexisting) source
-      for docs
-    - _However_: consider reading the recommended practices in this repository
-      and implementing as many of the best practices as you can. This groundwork
-      will pay big dividends later when you need to upgrade your practices and
-      update your documentation as an incubating project. Assistance is
-      available from CNCF TechDocs anytime, including answers to individual
-      questions or a documentation workshop.
-- **Incubating**
-  - [Website guidelines][]: all guidelines are satisfied.
-  - [Docs assessment][]: request an (re-)assessment through the CNCF [service
-    desk][]
-  - **Project documentation**:
-    - Stakeholders / personas are identified and their core needs (in terms of
-      docs) documented
-    - Hosted directly on the website (also see [Single-source requirement])
-    - Comprehensive, addressing most stakeholder needs
-- **Graduated**
-  - [Docs assessment][]: follow-through actions from any assessment are complete
-  - **Project documentation** fully addresses the needs of key stakeholders
-- **Archived**
-  - The website repo is in an [archived state][]
-  - The archived status of the project must be obvious to those visiting the
-    website, such as through the use of a prominent banner.
-  - If a successor project exists, link to its website and/or migration
-    documentation.
+maturity levels. So, for example, graduated projects must satisfy the
+requirements for incubating projects.
+
+We evaluate on the following:
+
+- Incubating: Are all [website guidelines][] satisfied? Graduated: Are all
+  guidelines satisfied?
+- Incubating: Was a [docs assessment][] requested through CNCF service desk?
+  Graduated: All follow-up actions addressed?
+- Incubating: User roles identified and doc needs documented? Graduated: All
+  stakeholder need identified?
+- Incubating: Website hosted directly? Graduated: Hosted directly?
+- Incubating: Is the [docs assessment][] comprehensive, addressing most
+  stakeholder needs? Graduated: Fully addresses needs of key stakeholders?
```

Comment:
I prefer the breakdown that we had before. @nate-double-u WDYT?

Reply:
I agree, I think we should keep this section structured as it was before, with additions as needed.